### ISSUE TITLE
Svg is not visible on physical device

### ISSUE THREAD

**Issue by Oxoproline** [luberda-molinet/FFImageLoading#1153]
### Description

I am using svg in my forms app. It is svg with linear gradient. When I debug app in simulator image is there and everything is working. When I debug/release the app in physical device(xiaomi MI A1 api27) image is not there, still other svgs without linear gradient are working fine on physical device, only this one is not shown. I tried to create png from the svg and it shows big array of bytes so I guess image is there it's just not visible. 

### Steps to Reproduce
svg
```
<?xml version="1.0" encoding="utf-8"?>
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 viewBox="0 0 1242 2688" style="enable-background:new 0 0 1242 2688;" xml:space="preserve">
    <g id="Layer_2" opacity="0.23">
        <linearGradient id="SVGID_1_" gradientUnits="userSpaceOnUse" x1="-491.9896" y1="2543.0627" x2="1165.5432" y2="2119.2195">
            <stop offset="0.2527" style="stop-color:#FFFFFF" />
            <stop offset="0.4516" style="stop-color:#ACA0A0" />
            <stop offset="0.7527" style="stop-color:#F1F2F2" />
            <stop offset="1" style="stop-color:#F1F2F2" />
        </linearGradient>
        <path class="st1" fill="url(#SVGID_1_)" d="M700.8,2119.6c-97.3,16.3-166.4,28.3-218.1,37.7c-40.2,7.3-73.8,13.6-102.5,19.4c-24.7,4.9-47.4,9.7-68.5,14.4
	c-24.9,5.5-47.5,10.9-68.6,16.2c-20.8,5.3-41.8,10.9-63.2,17c-23,6.5-48.5,14.2-77,23.2c-27,8.5-59.7,19.1-100.8,32.9v69.8
	c42.5-22.9,76-39.6,103.5-52.3c30-13.8,56.6-24.7,80.6-33.2c22.2-7.9,44.2-14.4,66.1-19.7c22.2-5.4,46.3-9.9,73.2-13.5
	c21.8-2.9,45.4-5.1,71.4-6.8c30.2-1.9,65.7-3.1,108.5-3.4c54.8-0.4,128.2,0.6,231.9,3.9c96.8,3.1,243,9,506.8,21.6v-216.2
	C951.2,2078.1,798.7,2103.2,700.8,2119.6z" />
        <linearGradient id="SVGID_2_" gradientUnits="userSpaceOnUse" x1="-541.0189" y1="2306.0676" x2="1113.6808" y2="1882.9487">
            <stop offset="0.2527" style="stop-color:#FFFFFF" />
            <stop offset="0.4516" style="stop-color:#ACA0A0" />
            <stop offset="0.7527"  style="stop-color:#F1F2F2" />
            <stop offset="1" style="stop-color:#F1F2F2" />
        </linearGradient>
        <path class="st2" opacity="0.10" fill="url(#SVGID_2_)" d="M456.2,2085c-36.8,15.4-68,27.8-95,37.9c-23.1,8.7-44.7,16.3-65,22.9c-22.6,7.4-43.6,13.6-63.4,18.9
	c-19.6,5.2-39.5,9.9-59.9,14.2c-21.8,4.6-46,9-72.9,13.1c-26.5,4.1-58.4,8.4-98,12.9v59.1c40.8-11.8,73.3-21,100.2-28.5
	c28.2-7.9,53.4-14.8,76.1-20.9c21.2-5.7,42-11.1,62.5-16.4c20.8-5.3,43.1-10.9,67.4-16.8c20.9-5.1,43.3-10.4,67.6-16.1
	c28.3-6.6,61.2-14.3,100.5-23.2c50.5-11.5,117.8-26.6,212.2-47.6c98.5-21.9,253.6-56.1,555.5-122.3v-245
	c-330.5,153.6-490.2,226.7-588.7,270.9C567.1,2037.7,503.9,2065.1,456.2,2085z" />
        <linearGradient id="SVGID_3_" gradientUnits="userSpaceOnUse" x1="-589.7419" y1="2103.8772" x2="1067.6346" y2="1680.0739">
            <stop offset="0.2527" style="stop-color:#FFFFFF" />
            <stop offset="0.4516" style="stop-color:#ACA0A0" />
            <stop offset="0.7527" style="stop-color:#F1F2F2" />
            <stop offset="1" style="stop-color:#F1F2F2" />
        </linearGradient>
        <path class="st3" fill="url(#SVGID_3_)" d="M99.4,2181.7c26.6-3.4,50.5-7.2,72.2-11.4c20.2-3.9,40-8.5,59.3-13.7c19.6-5.3,40.3-11.6,62.5-19.4
	c20.2-7,41.4-15.1,64.2-24.5c26.6-10.9,57.3-24.5,93.3-41.3c46.7-21.8,108.4-52.1,194.3-96c98.7-50.5,260.2-134.9,598.8-315.4
	v-287.7c-367.6,305.3-532.6,438.3-630.6,514.4c-79.5,61.8-137,103.7-180.9,133.4c-33.7,22.9-62.7,40.8-88.1,55
	c-21.7,12.1-42.2,22.3-61.9,30.6c-20.8,8.9-40.3,15.7-59,20.9c-18.5,5.1-37.5,9.1-57,11.9c-20.8,3-43.7,4.8-69.2,5.3
	c-26,0.5-57.2-0.4-95.5-3.5v50.8C41.3,2188.2,73,2185.1,99.4,2181.7z" />
    </g>
</svg>
```

### Expected Behavior

Svg will show on real device

### Actual Behavior

Svg with linear gradient is not visible on physical device.

### Basic Information

- Version with issue: 2.4.3.840
- Last known good version: none
- Platform: Android, iOS is fine so far

**Comment by EmilAlipiev** [comment]
can you provide a test project please?

**Comment by daniel-luberda** [comment]
I'm sure that's a Xamarin's linker issue: https://xamarinhelp.com/xamarin-linker/

**Comment by Oxoproline** [comment]
I don't think it is linker issue. It works in emulator and I think in debug linker is disabled. 
Yeah it might be SkiaSharp bug. When I will have time, @EmilAlipiev I will create sample app and test it on skia itself and post it here.

**Comment by hashmiaj** [comment]
I'm having the same issue! The image is loading on my iPhone but not on my android. I have the loading image up with the svg but once the loading is complete, the loading image disappears and it is just blank white no svg.

**Comment by EmilAlipiev** [comment]
@hashmiaj did you try without loading image? does it work if you dont set it?

**Comment by hashmiaj** [comment]
I've solved the solution on my end. The problem was that I was putting it in a container and had to set the width and height smaller than what I had for iOS. So I had svg.widthRequest = 2000 and svg.heightRequest = 2000 for iOS and I changed it to 1000 for both width and height for android and it displays and works fine. There is however a problem with android and zooming out but that is for another thread :)
